### DISCUSSION TITLE
"The chathistory is too long, please create a new chat or summarize it."

### DISCUSSION THREAD

**Discussion by extromedia** [nicobrauchtgit/obsidian-Smart2Brain#168] [Q&A]
Why do I receive this message after only a short chat (about 3 questions) and what can I do about it?
"The chathistory is too long, please create a new chat or summarize it."
Thank you.

I'm using Ollama with llama2 (nomic-embed-text), creativity is 5%, similarity is 25%.

Any advice is greatly appreciated.

**Comment by Leo310** [comment]
I know Its been some time, but it will be fixed in the next Release ;)
Wont take long anymore!
